src/src/Palindrom.java

```diff
@@ -0,0 +1,16 @@
+public class Palindrom {
```

Comment:
It seems that you haven't done guess the word game, so the mark will be lowered for that.